README.md

```diff
@@ -24,6 +24,11 @@ Can review bounties in exchange for rewards.
 
 Reviewers claim reviewable tasks and complete a review on the task assigning it a state of pass, fail or retry. Upon review completion the reviewer is awarded their share of the award amount as detailed by the bounty spec.
 
+#### Eligibility for Reviewers
+In order to provide code reviews of the highest possible quality, anybody who wants to review bounties must be an established member within our [Discord server](https://discord.gg/jDjSQvTHTC).
```

Comment:
Discord links apparently expire after a week by default. Would there be reasons not to include a link in the readme?